bigtable/google/cloud/bigtable/instance.py

```diff
@@ -329,8 +329,8 @@ def list_tables(self):
         for table_pb in table_list_pb:
             table_prefix = self.name + '/tables/'
             if not table_pb.name.startswith(table_prefix):
-                raise ValueError('Table name %s not of expected format' % (
-                    table_pb.name,))
+                raise ValueError('Table name {} not of expected format'
+                    .format(table_pb.name,))
```

Comment:
`flake8` doesn't think the indentation is correct here.  I think it would be appeased by:

```python
                raise ValueError(
                    'Table name {} not of expected format'.format(
                        table_pb.name))
```